cmd/helm/history.go

```diff
@@ -19,7 +19,7 @@ package main
 import (
 	"fmt"
 	"io"
-	"time"
+	stdtime "time"
```

Comment:
Would it make sense to call helm's time package `helmtime` and keep `time` as `time`? I feel that may be less confusing to the reader that `stdtime` isn't some other third party package. Also, it opens up the opportunity for us to say

> it's `helmtime`

:trollface:

Reply:
I thought about that. Since you brought it up, I'll change it

Reply:
Done